docs/CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix external sync order in serializers (OSIDB-3029)
 - Make Taskman service validate Jira token (OSIDB-2203)
 
+### Changed
+- special_handling_flaw_missing_cve_description Alert to
+  special_consideration_flaw_missing_cve_description (OSIDB-2955)
+- special_handling_flaw_missing_statement Alert to
+  special_consideration_flaw_missing_statement (OSIDB-2955)
```

Comment:
wrong section

Reply:
I don't think it really matters here as border between fixed and changed is very thin here. The fix of the validation was more about completely changing it to something else.

Reply:
Sorry for not being clear enough - you are updating `4.1.1.`

Reply:
Aha! Thanks :smile: